### ISSUE TITLE
Learning Rate Decay

### ISSUE THREAD

**Issue by magic282** [JianGoForIt/YellowFin_Pytorch#18]
Hi,

For YellowFin optimzer, do I need to use the learning rate decay trick?
For example, I evaluate the model on dev set and if the performance drops I will halve the learning rate.
This trick works very well for optimizers such as Adam and Adadelta. So will it also work if I switch to the YellowFin optimizer?

Thanks.

**Comment by JianGoForIt** [comment]
Hey @magic282 

Good question! We have this feature in the repo. Please browse to the Readme section "Interface for manual finer control". And there is an example for it here. https://github.com/JianGoForIt/YellowFin_Pytorch/blob/master/pytorch-cifar/main.py#L109

Cheers

**Comment by JianGoForIt** [comment]
Please open it again if necessary.
